lib/core-integration/src/Test/Integration/Scenario/API/Transactions.hs

```diff
@@ -1284,7 +1301,7 @@ spec = do
                         ]
                     , assertions =
                         [ expectListSizeEqual 1
-                        , expectListItemFieldEqual 0 amount a2
+                        , expectListItemFieldEqual 0 DSL.amount a2
```

Comment:
All these `DSL` seem wrong. I understand that it's because of the imported constructor, but we shouldn't have to import the tx constructor here :no_mouth:

Reply:
indeed, corrected